integration/fixtures/forseti_integration_testing/scripts/wait-for-forseti.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Block until the Forseti startup script has finished running.
+
+echo "Waiting for up to 360 seconds for Forseti to be ready."
+
+for _ in {1..360}; do
+  if [[ -f /etc/profile.d/forseti_environment.sh ]]; then
+    echo "Forseti is ready."
+    echo "Waiting for startup script to complete"
+    sleep 60
+    exit 0
+  else
+    sleep 1
+  fi
+done
+
+echo "Forseti was not ready after 360 seconds!"
```

Comment:
Since `360` is used several times, can you put it into a variable, so that it's easy to change in the future?

Reply:
Done.